Octokit/Helpers/HttpClientExtensions.cs

```diff
@@ -7,16 +7,12 @@ namespace Octokit
 {
     public static class HttpClientExtensions
     {
-        /// <summary>
-        /// OBSOLETE
-        /// </summary>
-        [Obsolete("This will be removed in a future release")]
-        public static Task<IResponse<T>> Send<T>(this IHttpClient httpClient, IRequest request)
+        public static Task<IResponse> Send(this IHttpClient httpClient, IRequest request)
```

Comment:
:thumbsup: to lazy, that work hasn't made it into a GHfW PR yet.

If I get caremad about it again, I'll PR it separately.